handler/instance_started.go

```diff
@@ -118,7 +118,12 @@ func (h *InstanceStarted) Handle(ctx context.Context, workerID int, msg kafka.Me
 		"num_variables": len(resp.Dataset.Variables.Edges),
 	})
 
-	ireq := h.createUpdateInstanceRequest(ctx, resp.Dataset.Variables, e, r.CantabularBlob, i.CodeLists)
+	if len(i.Editions) < 1 {
+		return errors.New("no editions found in instance")
+	}
+	edition := i.Editions[0] // for census, we assume there will only ever be one
```

Comment:
If there is only one, why do we have an array/slice?
Is this struct being shared by other business logic?

I appreciate that this may be legacy code. I am trying to understand the business logic as well :)

Reply:
There can be more than one edition, and the logic may change in the future. For now Fran has said the we should use the first element of the array.